### PR TITLE
Array - pascal triangle

### PR DESCRIPTION
**Pull request by DevratAgarwal**
## Checklist
- Solution is tested on sphere engine <br/>
    `Note:` Please copy only the solution part. Do not copy/paste the template code. That way, any issues with template code can be caught early on.
    - [ ] Author
    - [x] Reviewer
- PR title is correct - [PR title conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#title-for-a-pull-request)
    - [ ] Author
    - [x] Reviewer
- Folder/File naming is correct - [Naming conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#naming-of-files-and-folders)
    - [ ] Author
    - [x] Reviewer
- Coding conventions are followed - [Coding conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#coding-conventions)
    - [ ] Author
    - [x] Reviewer
- Test cases (action item for **reviewer**) <br/>
    - To be ensured by checking the problem on sphere-engine
    - If any test cases are missing, please add them - [how to add new test cases](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#adding-new-test-cases)
    - [x] There are at least 2 test cases
    - [x] There is at least 1 test case that is different from the sample test cases

### CODE REVIEW

**Comment by sailesh-ramayanam** [comment]
@satya-9 
1. Write a function that takes an integer value n as input and prints ~first n lines~ `n`th line of the Pascal’s triangle.
2. Please format `n`, wherever it appears
3. `n` is written in lower case in the description and in upper case in input/output format. Please use same case everywhere.

File: solutions/arrays/pascal-triangle.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+let rowNumber = parseInt(readLine())
+if(rowNumber ===1)
+{
+	console.log(1);
+}
+else
+{
+	let arr=[1]
```
Comment (sailesh-ramayanam) [inline solutions/arrays/pascal-triangle.js:20]: Rename `arr` to `prevRow`

File: solutions/arrays/pascal-triangle.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+	for(let row=2;row<=rowNumber;row++)
+	{
+		let temporaryArray=[]
```
Comment (sailesh-ramayanam) [inline solutions/arrays/pascal-triangle.js:23]: Rename `temporaryArray` to `currRow`

File: solutions/arrays/pascal-triangle.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+		for(let column=1;column<=row;column++)
+		{
+			if(column===1)
```
Comment (sailesh-ramayanam) [inline solutions/arrays/pascal-triangle.js:26]: Always use curly braces , even for single line body. Apply this wherever applicable in  this program as well as in your other PRs.

**Review by sailesh-ramayanam — CHANGES_REQUESTED** [review]
@DevratAgarwal 
1. Please remove other files from this PR
2. Please address my comments below
3. Some of the comments are applicable to your other PRs as well. Please make changes wherever applicable.

**Comment by DevratAgarwal** [comment]
@sailesh-ramayanam @satya-9 changes are addressed

**Comment by LVSsandeepVarma** [comment]
@sailesh-ramayanam  updated the problem description

**Review by LVSsandeepVarma — APPROVED** [review]

**Comment by prathyusha10x** [comment]
@sailesh-ramayanam please review the updated code and approve the PR by today EOD

**Review by sailesh-ramayanam — APPROVED** [review]

**Pull request merged**